jira/exceptions.py

```diff
@@ -17,6 +17,7 @@ def __init__(self, status_code=None, text=None, url=None, request=None, response
         self.response = response
         self.headers = kwargs.get('headers', None)
         self.log_to_tempfile = False
+        self.travis = False
```

Comment:
This is a separate bug-fix due to the `.travis` field being initialized outside the constructor (violation PEP 8).